bucket/resvg.json

```diff
@@ -1,16 +1,16 @@
 {
-    "version": "0.27.0",
+    "version": "0.29.0",
     "description": "An SVG rendering library.",
     "homepage": "https://github.com/RazrFalcon/resvg",
     "license": "MPL-2.0",
     "architecture": {
         "64bit": {
             "url": [
-                "https://github.com/RazrFalcon/resvg/releases/download/v0.27.0/resvg-win64.zip",
+                "https://github.com/RazrFalcon/resvg/releases/download/v0.29.0/resvg-win64.zip",
                 "https://github.com/RazrFalcon/resvg/releases/download/v0.27.0/usvg-win64.zip"
```

Comment:
Please remove this usvg too, as the version is mismatched.

Reply:
done